packages/react-tweet/src/twitter-theme/tweet-author-verified-badge.tsx

```diff
@@ -0,0 +1,34 @@
+import clsx from 'clsx'
+import s from './tweet-author-verified-badge.module.css'
+import { Verified, VerifiedBusiness, VerifiedGovernment } from './icons'
```

Comment:
When running the dev apps, the `create-react-app` app fails to build because it can't find this file to import I believe you'll want to import from `./icons/index.js` file here.